keymanager-runtime/src/main.rs

```diff
@@ -47,6 +37,9 @@ fn main() {
                 _rpc_demux: &mut RpcDemux,
                 rpc: &mut RpcDispatcher,
                 _txn: &mut TxnDispatcher| {
+        // Initialize the set of trusted policy signers.
+        init_trusted_policy_signers();
+
         // Register RPC methods exposed via EnclaveRPC to remote clients.
```

Comment:
Below (and `crate::methods::*`) seems like a lot of boilerplate that each key manager runtime will need anyway if it only wants to change the set of policy signers? Consider moving them to the common crate.

Reply:
I did initially consider moving this to the api-common crate, but decided against it, since it's more flexible this way.  New key managers might want to run other code during initialization and/or add new API endpoints.

I got the impression that this is something we'd eventually want to support, so there's no point in moving this into the common crate now, only to move it back a few weeks later :)

Reply:
Ok, let's leave it like this for now.